### PR TITLE
test: Probe DRM robustness levels

### PR DESCRIPTION
**Pull request by joeyparrish**

### CODE REVIEW

**Comment by avelad** [comment]
@joeyparrish the robustness can be different for audio and video, so we need differentiate it

**Comment by joeyparrish** [comment]
It now probes and reports audio and video robustness separately.  I merged main into this branch, too, so you should now see the results of these probes in the CI logs for this PR.

**Comment by avelad** [comment]
@shaka-bot test

**Review by avelad — APPROVED** [review]

File: lib/media/drm_engine.js [hunk 1]
```diff
@@ -1807,13 +1807,40 @@ shaka.media.DrmEngine = class {
...
+      '150',
+      '2000',
+      '3000',
+    ];
+
+    const testRobustness = {
+      'com.widevine.alpha': widevineRobustness,
+      'com.microsoft.playready': playreadyRobustness,
+      'com.microsoft.playready.recommendation': playreadyRobustness,
+      'com.chromecast.playready': playreadyRobustness,
```
Comment (tykus160) [inline lib/media/drm_engine.js:1828]: Only recommendation key system supports checking robustness. Legacy key systems ignore it, so to check SL3000 you need to use `com.microsoft.playready.hardware` key system instead.

Comment (avelad) [reply lib/media/drm_engine.js:1828]: Actually that only applies to Edge and Xbox, right?

Comment (tykus160) [reply lib/media/drm_engine.js:1828]: Recommendation is available for sure on Edge, Xbox and PS5.

Comment (avelad) [reply lib/media/drm_engine.js:1828]: Ok, please create a PR with this change. Thanks!

**Pull request merged**